pkg/extensions/sync/content.go

```diff
@@ -240,6 +247,32 @@ func filterTagsByRegex(tags []string, regex string, log log.Logger) ([]string, e
 	return filteredTags, nil
 }
 
+// excludeTagsByRegex filter-out images by tag regex given in the config.
+func excludeTagsByRegex(tags []string, regex string, log log.Logger) ([]string, error) {
+	if len(tags) == 0 || regex == "" {
+		return tags, nil
+	}
+
+	filteredTags := make([]string, 0, len(tags))
+
+	log.Info().Str("excludeRegex", regex).Msg("filtering out tags using regex")
```

Comment:
Make this log.Debug()

Reply:
This should be the same as the message at line 232.
If would be very confusing for users who disabled debug logs to see only that message.
Either we keen both info, or both debug.